qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/CreateIdentityUITest.java

```diff
@@ -68,28 +75,50 @@ public void setUp() throws Exception {
         wakeLock = UIActionHelper.wakeupDevice(mActivity);
         mSystemAnimations = new SystemAnimations(mActivity);
         mSystemAnimations.disableAll();
+
+        //TODO TMP Fix option
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            InstrumentationRegistry.getInstrumentation().getUiAutomation().executeShellCommand(
+                    "pm grant " + InstrumentationRegistry.getTargetContext().getPackageName()
+                            + " android.permission.READ_PHONE_STATE");
+        }
     }
 
 
     @Test
     public void testCreateIdentity() throws Throwable {
-        String identity = "spoon2";
-        createIdentityPerformEnterName(identity);
+        createIdentityPerformEnterName();
+        createIdentityEnterEmail();
+        createIdentityEnterPhone();
         createIdentityPerformConfirm();
     }
 
-    private void createIdentityPerformEnterName(String identity) throws Throwable {
+    private void createIdentityPerformEnterName() throws Throwable {
         onView(withText(R.string.create_identity_create)).check(matches(isDisplayed())).perform(click());
-        onView(allOf(withClassName(endsWith("EditTextFont")))).perform(setText(identity), pressImeActionButton());
         onView(withText(R.string.create_identity_enter_name)).check(matches(isDisplayed()));
-        closeSoftKeyboard();
-
         UITestHelper.screenShot(UITestHelper.getCurrentActivity(mActivity), "input");
+        onView(withId(R.id.et_name)).perform(setText(name));
```

Comment:
who's ET? ;)
This abbreviation decreases readability. If it means edit_text then `edit_test_name` is probably preferrable. Id doesn't mean much to me either but at least one does not have do decrypt the abbreviation AND the ids' name